test/companies/schema/company_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Companies.Schema.CompanyTest do
+  @moduledoc """
+    This module holds the unit tests to the company changeset
+  """
+  use Companies.DataCase
+
+  alias Companies.{
+    Schema.Company
+  }
+
+  @valid_company_params %{
+    blog: "Company blog",
+    description: "Description of the company",
+    github: "github account of the company",
+    location: "Amsterdam",
+    name: "Company name",
+    url: "link of the company",
+    industry_id: 12346
+  }
+  describe "company changeset" do
+    test "company/2 validates with the correct parameters" do
```

Comment:
can you add an empty line between the `@valid_company_params` and the `describe` please?

Can you

Reply:
Yes I can